### DISCUSSION TITLE
Multiselect max displayed selected items prop

### DISCUSSION THREAD

**Discussion by ibadkhan1996** [mantinedev/mantine#6283] [Feature requests]
Hi Vitaly! I saw that you have added example of this on examples page. But I really want it on prop based. Like if I want to limit the selected items to display there should be a prop of maxDisplaySelectedItems and as I would pass integer it would limit to that number. can you please add this feature in next release. Thanks!

**Comment by rtivital** [comment]
No, it is not planned to be added, this feature is available via combobox if you need this in your app.
